sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/MicrosoftTeamsUserIdentifier.java

```diff
@@ -8,28 +8,14 @@
 /**
  * Communication identifier for Microsoft Teams User
  */
-public class MicrosoftTeamsUserIdentifier extends CommunicationIdentifier {
+public final class MicrosoftTeamsUserIdentifier extends CommunicationIdentifier {
 
     private final String userId;
     private final boolean isAnonymous;
     private CommunicationCloudEnvironment cloudEnvironment = CommunicationCloudEnvironment.PUBLIC;
 
     private String rawId;
 
-    /**
```

Comment:
Any reason this has to be removed?

Reply:
There's an apiview comment saying that this constructor isn't needed as users can just chain the cloudEnvironment using the set function we already have.